lib/collection/request-auth/oauth1.js

```diff
@@ -103,16 +105,14 @@ module.exports = {
                 system: true
             });
         }
+        else if (_.isString(request.method) && request.method.match(MATCH_PUTPOST) &&
```

Comment:
Not necessary to refactor code here. Revert this, and add a todo if you think this needs a change. We'll take it up in a future PR.